template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java

```diff
@@ -114,6 +114,11 @@ public static boolean isRazConfigurationSupportedInDatahub(ClouderaManagerRepo c
         return isVersionNewerOrEqualThanLimited(clouderaManagerRepoDetails::getVersion, CLOUDERAMANAGER_VERSION_7_2_2);
     }
 
+    public static boolean isRazTokenConfigurationSupported(ClouderaManagerRepo clouderaManagerRepoDetails) {
+        LOGGER.info("ClouderaManagerRepo is compared for Raz Ranger token support in Data Hub and Data Lake");
+        return isVersionNewerOrEqualThanLimited(clouderaManagerRepoDetails::getVersion, CLOUDERAMANAGER_VERSION_7_4_2);
```

Comment:
The Jira says the corresponding `ranger.raz.bootstrap.servicetypes` property depends on CR 7.2.10. If so, why are we checking for the CM version here?

Reply:
the cm version is 7.4.2 for CDH 7.2.10